makefile

```diff
@@ -0,0 +1,14 @@
+
+VOLUMES := -v $(CURDIR):/src -v $(CURDIR)/public:/target 
+
+build:
+	docker run --rm -it $(VOLUMES) klakegg/hugo:0.53-ext -D -v
```

Comment:
Can we also use `klakegg/hugo:0.53-ext-alpine` within the makefile, which is a much smaller Docker image (and therefore faster to download, takes less disk space, etc...)?
Maybe we can even set this as a variable on top of the makefile.

Reply:
Thank you for the input - applied all of it.